packages/next/src/server/ReactDOMServerPages.js

```diff
@@ -0,0 +1,20 @@
+let ReactDOMServer
+
+try {
+  ReactDOMServer = require('react-dom/server.edge')
```

Comment:
This still produces warnings in Turbopack and Webpack.

Webpack we ignore via `ignoreWarnings`. Turbopack's seems to not surface to users.

Reply:
Will try to use aliases in externals in a follow-up depending on which React version we're compiling for.

Reply:
https://github.com/vercel/next.js/pull/70463